### ISSUE TITLE
 nativeAppChecker: WebLoginOnly() - crashes

### ISSUE THREAD

**Issue by eugenCodreanu** [tonyli508/LinkedinSwift#50]
Hi, I found this  line of code "nativeAppChecker: WebLoginOnly()", but then I try to add in my app and launch on device, it crashes.

**Comment by tonyli508** [comment]
@eugenCodreanu did you try the demo app yet?

**Comment by eugenCodreanu** [comment]
Yes. I launch the Example project. Insert all my linked in details. On press login it swith to webpage, but nothing appear, only Cancel button on top left corner. After 1 sec it crash,

WebThread (11): EXC_BAD_ACCESS (code=1, address=0x2ffffff6b)

![Screen Shot 2019-05-02 at 10 52 40 AM](https://user-images.githubusercontent.com/22447099/57062410-b3f5cc00-6cc8-11e9-9822-a3f9c6044746.png)

**Comment by tonyli508** [comment]
@eugenCodreanu I updated profile request to version 2, I cannot reproduce the crash, can you try?

**Comment by jondoes** [comment]
@tonyli508 
I also facing the same issue. I run the demo code with my Iphone.
Device Specification:
Model: Iphone 6
OS: 10.0

But If I run this demo code with Iphone 7+ with 12.x.x OS.
And it works fine.

Please help me.
